packages/core/src/components/Calendar/DayPicker/DayPickerItem.js

```diff
@@ -91,6 +93,9 @@ export default class DayPickerItem extends Component<Props> {
       state.isFirstHighlighted ? css.firstHighlighted : null,
       state.isLastHighlighted ? css.lastHighlighted : null,
       state.isClosed ? css.closed : null,
+      state.isGreen ? css.green : null,
```

Comment:
is it worth labelling these with as the business case rather than the colour? i.e `state.isAvailable ? css.green : null` 
might help to give bit a more context about what the colours are actually used for

Reply:
I wouldn't as this will tie the library more to our website

Reply:
Yeah, I had this same debate with myself. But decided Loran's way.